pandas/api/tests/test_api.py

```diff
@@ -61,14 +61,14 @@ class TestPDApi(Base, tm.TestCase):
                           'SparseTimeSeries', 'Panel4D',
                           'SparseList']
 
-    # these should be deperecated in the future
+    # these should be deprecated in the future
     deprecated_classes_in_future = ['Term', 'Panel']
 
     # these should be removed from top-level namespace
     remove_classes_from_top_level_namespace = ['Expr']
 
     # external modules exposed in pandas namespace
-    modules = ['np', 'datetime', 'datetools']
+    modules = ['np', 'datetime', 'sys', 'warnings']
```

Comment:
It's not helpful because I don't know how to proceed with it.  Sure, you did provide comments below that provide further guidance, but in isolation, "fixing it" is far from clear.